README.md

```diff
@@ -295,6 +296,56 @@ clip-retrieval-front 3005
 
 You can also run it with `clip-retrieval front` or back from the python package.
 
+## Clip client
+
+Clip client is a simple python module that allows you to query the backend remotely.
+
+```python
+from clip_retrieval.clip_client import ClipClient, Modality
+
+client = ClipClient(
+    url="https://knn5.laion.ai/knn-service",
+    aesthetic_score=9,
+    aesthetic_weight=0.5,
+    modality=Modality.IMAGE,
+    num_images=40,
+    num_result_ids=3000,
+)
+```
+
+### Query by text
+
+```python
+results = client.query(text="an image of a cat")
+results[0]
+> {'url': 'https://finalmonsoon.files.wordpress.com/2017/10/kitten.jpg?w=404&', 'caption': 'kitten', 'id': 5098117701, 'similarity': 0.9367108941078186}
```

Comment:
That similarity is not possible with text/image, did you really query with text here (and not image) ?

Reply:
I was confused by this as well actually - let me re-run this and double-check I'm not query via image - is the similarity metric returned not the original clip score in either case?

Reply:
so yeah i verified it was from an image search - why is that? in any case - i changed this to a contrived example instead.